pages/transcode.md

```diff
@@ -0,0 +1,23 @@
+title: Trans*Code
+---
+
+[Trans*Code](http://trans-code.org/) returns to PyCon UK for a day-long
+hackathon to address issues facing the trans\* and nonbinary community.  The
+day is open to all trans\* and nonbinary people and allies, and we welcome
+participants from all skill levels and backgrounds.
+
+The day will begin with participants suggesting apps and projects that are of
+particular interest to the trans\* and nonbinary community.  We'll then
+self-organize into teams, and spend the day working on our projects.
+
+***
+
+Entrance to the Trans\*Code hack day is included in the [conference
+ticket](/tickets/).
+
+However, if you are just coming to Cardiff for the hack day on Monday,
+[tickets](https://www.eventbrite.com/e/transcode-pycon-uk-2017-tickets-38377166137?aff=utm_source%3Deb_email%26utm_medium%3Demail%26utm_campaign%3Dnew_event_email&utm_term=eventurl_text)
+are free.
+
+We are also able to give away a number of tickets to the whole conference to
+Trans\*Code attendees. Please, see instructions on the EventBrite event
```

Comment:
nit: this needs a full stop

It also refers to an Eventbrite event (nit: no uppercase “B”), but that’s not mentioned elsewhere in the text – it’s obscured by the URL text. Maybe rewrite the penultimate paragraph to be:

    However, if you are just coming to Cardiff for the hack day on Monday, tickets are
    free. Book your ticket [on Eventbrite](…).